.github/workflows/release.yml

```diff
@@ -347,18 +319,31 @@ jobs:
           echo "Metabase OSS: image is going to be pushed to ${{ github.repository_owner }}/metabase"
           echo "DOCKERHUB_REPO=${{ github.repository_owner }}/metabase" >> $GITHUB_ENV
         fi
-
+    - uses: actions/download-artifact@v4
+      name: Retrieve previously downloaded Uberjar
+      with:
+        name: metabase-${{ matrix.edition }}-uberjar
+    - name: Move the Uberjar to the context dir
+      run: mv ./metabase.jar bin/docker/.
     - name: Login to Docker Hub
       uses: docker/login-action@v2
       with:
         username: ${{ secrets.DOCKERHUB_RELEASE_USERNAME }}
         password: ${{ secrets.DOCKERHUB_RELEASE_TOKEN }}
-    - name: Retag and push container image to Docker Hub
-      run: |
-        echo "Pushing ${{ steps.canonical_version.outputs.result }} to ${{ env.DOCKERHUB_REPO }} ..."
-        docker tag localhost:5000/local-metabase:${{ steps.canonical_version.outputs.result }} ${{ env.DOCKERHUB_REPO }}:${{ steps.canonical_version.outputs.result }}
-        docker push ${{ env.DOCKERHUB_REPO }}:${{ steps.canonical_version.outputs.result }}
-        echo "Finished!"
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v3
+    - name: Build ${{ matrix.edition }} container
+      uses: docker/build-push-action@v6
+      with:
+        context: bin/docker/.
+        platforms: linux/amd64,linux/arm64
```

Comment:
I'm not seeing this change in the files now, did you remove them?

Reply:
oh yes, I updated this to remove the multi-arch bit of this and just make the other changes

1) the underlying issue that prevented running on Java 21 isn't merged yet, and it's unclear if it will get backported to v51
2) I don't want to actually build the arm image until we can also test it, which I didn't figure out quite yet